pkg/cmd/secrets.go

```diff
@@ -139,6 +139,15 @@ JSON Secret: "{\"logging\": \"info\"}"`,
 	Run:  substituteSecrets,
 }
 
+var secretsEditCmd = &cobra.Command{
+	Use:     "edit",
+	Short:   "Edit secrets in your editor",
+	Long:    "Shows a list of current secrets in the config and allows you to edit in your $EDITOR",
+	Example: `$ doppler secrets edit`,
+	Args:    cobra.NoArgs,
+	Run:     editSecrets,
+}
```

Comment:
Does this gracefully handle secret renames? If not we should just add a note about that limitation to the help text.